wolfssl/internal.h

```diff
@@ -5387,6 +5387,11 @@ WOLFSSL_LOCAL int oid2nid(word32 oid, int grp);
 WOLFSSL_LOCAL word32 nid2oid(int nid, int grp);
 #endif
 
+#ifdef WOLFSSL_DTLS
+WOLFSSL_API int wolfSSL_DtlsUpdateWindow(word16 cur_hi, word32 cur_lo,
```

Comment:
why this can't be WOLFSSL_LOCAL?

Reply:
WOLFSSL_LOCAL isn't exported to applications using the library. The unit test is an application. It is in internal.h, which isn't installed with `make install`.

Reply:
Oh, interesting. I think in the future it would beneficial if we can unit testing `WOLFSSL_LOCAL` functions somehow